Add `LC_ALL` and `LANG` export for Amazon Linux 2 (#899)
DataDog/chef-datadog: README.md

5. Wait for the next scheduled `chef-client` run or trigger it manually.

#### Troubleshooting

> On some OS (for example Amazon Linux 2) you might have the following error during installation:

Reviewer: ```suggestion
On some operating systems such as Amazon Linux 2, you may encounter the following error during installation:
```

```
ValueError: unknown locale: UTF-8
```

You can fix this by running:
```
export LC_ALL=en_US.UTF-8
export LANG=en_US.UTF-8
```

> You can now restart the installation and it should be fine.

Reviewer: ```suggestion
Then, restart the installation.
```
